packages/rocketchat-livechat/client/views/sideNav/livechat.js

```diff
@@ -36,6 +36,29 @@ Template.livechat.helpers({
 			}
 		});
 	},
+	inquiries() {
+		var user = Meteor.user();
+
+		// get all inquiries of the department 
+		var inqs = LivechatInquiry.find({
+			agents: user._id,
+			status: 'open'
+		}, {
+			sort: {
+				'ts' : 1
+			}
+		});
+
+		// for notification sound 
+		inqs.forEach(function (inq) {
+			KonchatNotification.newRoom(inq.rid);
+		});
+
+		return inqs;
+	},
+	guestPool() {
+		return RocketChat.settings.get('Livechat_Routing_Method') == 'Guest_Pool';
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected '===' and instead saw '=='.](https://www.codacy.com/app/RocketChat/Rocket-Chat/file/2726498730/issues/source?bid=3199634&fileBranchId=3348698#l60)